### ISSUE TITLE
Regarding Generalized IOU Loss

### ISSUE THREAD

**Issue by samyak0210** [CoinCheung/pytorch-loss#7]
Can G-IOU be used as a loss function? Actually I am training a bounding box prediction model and using GIOU as a loss function but after some few iterations of the first epoch of training, I am getting constant values.

**Comment by CoinCheung** [comment]
Yes, in the object detection field, and especially in these days, there are many paper exploring using iou-based loss for bounding box regression. The giou loss for object detection was proposed earlier, and you can find the paper [here](https://arxiv.org/abs/1902.09630).

**Comment by CoinCheung** [comment]
I am closing this since you might have figured it out.
